messenger/http_transporter.go

```diff
@@ -85,10 +85,10 @@ func (s *stoppedState) Recv() (*Message, error)                        { return
 func (s *stoppedState) Inject(ctx context.Context, msg *Message) error { return errTerminal }
 func (s *stoppedState) Stop(graceful bool) error                       { return errTerminal }
 func (s *stoppedState) Install(messageName string)                     {}
-func (s *stoppedState) Start() <-chan error {
+func (s *stoppedState) Start() (upid.UPID, <-chan error) {
 	ch := make(chan error, 1)
 	ch <- errTerminal
-	return ch
+	return upid.UPID{}, ch
```

Comment:
I'm not familiar with this code. Why are we returning an empty `UPID` here and the other implementations of this interface below?